src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/extensions/BitbucketPostBuildStatus.java

```diff
@@ -0,0 +1,45 @@
+package com.atlassian.bitbucket.jenkins.internal.scm.extensions;
```

Comment:
I think this class needs some tests as well.. what happens if there is more than one buildData? (can there be more than one?) etc

Reply:
I investigated this. The `getLastBuild` method verifies the builds to ensure only one is returned and I believe all jobs in Jenkins are run sequential, so I don't think there is a danger of multiple build data.